docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -27,16 +31,29 @@ Vulnerability processing happens on the Fleet instance and not on the host machi
 detected vulnerabilities cannot be used in the same way you would use an osquery query (e.g. you wouldn't
 be able write a query to retrieve all CVEs).
 
+### Linux hosts
+
+First, we determine what Linux distributions are part of your fleet (keep in mind that there will
+be a small delay between the time a new Linux hosts is added and the time the host is 'detected'). We then
+use
+that information to determine what OVAL definitions need to be downloaded and parsed - you can find
+a list of all the OVAL definitions we use
+[here](https://github.com/fleetdm/nvd/blob/master/oval_sources.json). OVAL definitions will be
+refreshed on a daily basis.
+
+Finally, we look at the software inventory of each host and execute the assertions contained in the
+corresponding OVAL file - any match is reported using the same channels as with Windows/Mac OS vulnerabilities
+
 ## Coverage
 
-Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
+For Windows/Mac OS Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
 
 ### Apps
 - macOS
 - Windows
 
 ### Browser plugins
-- macOS, Windows, Linux
+- macOS, Windows
```

Comment:
Not sure I understand why this item is in here. Does it mean "we only check for plugins in macOS and Windows"?

Reply:
That's a very good question!

OVAL definitions only test things installed via the package manager (rpm, apt, etc) - so, at least on Ubuntu, we will be only targeting a small subset of those - if we want to scan for browser plugins on Linux using the NVD dataset we just need to change [this](https://github.com/fleetdm/fleet/blob/main/server/vulnerabilities/cve.go#L116) 
From:
```
CPEs, err := ds.ListSoftwareCPEs(ctx, oval.SupportedHostPlatforms)
```

To:
```
// Exclude Linux package sources: rpm, deb sources
CPEs, err := ds.ListSoftwareCPEs(ctx, oval.SupportedSources)
```